rdr_service/dao/participant_summary_dao.py

```diff
@@ -586,11 +588,13 @@ def calculate_enrollment_status(
                 and (gror_consent == QuestionnaireStatus.SUBMITTED or consent_cohort != ParticipantCohort.COHORT_3)
             ):
                 return EnrollmentStatus.FULL_PARTICIPANT
-            return EnrollmentStatus.MEMBER
+            if consent_expire_status != ConsentExpireStatus.EXPIRED:
+                return EnrollmentStatus.MEMBER
         return EnrollmentStatus.INTERESTED
 
     def calculate_member_time(self, consent, participant_summary):
-        if consent and participant_summary.enrollmentStatusMemberTime is not None:
+        if (consent or participant_summary.ehrConsentExpireStatus == ConsentExpireStatus.EXPIRED) and \
```

Comment:
Before this change, any participants that explicitly refused EHR consent having having a consent would have their member time cleared. Now if a participant explicitly refuses EHR consent after they have an expiration this will still give them an enrollmentStatusMemberTime. Is that desired behavior?

Reply:
Ah, good catch! I thought even it's expired, we still need the `enrollmentStatusMemberTime` for the participant count over time, but you are right, we didn't do that for the withdrawal participants. I will change this back to keep consistent. I will also check with Katie about the requirement for the participant count over time dashboard, if they need, we can do a fix for that.